File: moban_java/src/main/java/org/jdk/project/dto/config/DeviceDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DeviceDto {
  private Long id;
  private String deviceName;
  private String deviceType;
  private String deepGlintDeviceId;
  private String roomCode;
  private String deptName;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/DeviceDto.kt
package org.jdk.project.dto.config

data class DeviceDto(
    var id: Long? = null,
    var siteId: Long? = null,
    var siteCode: String? = null,
    var siteName: String? = null,
    var deviceName: String? = null,
    var deviceType: String? = null,
    var deepGlintDeviceId: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
